kwok/test_member_namespace_map.yml

```diff
@@ -0,0 +1,15 @@
+apiVersion: v1
```

Comment:
please fix the pre-commit comment, and also add some namespaces that are not RHTAP workspaces to make sure the script is not returning them.

Also try to make the data look more like what we really have in RHTAP - where workspaces end with the `-tenant` prefix and the owner username can be different.

Reply:
fixed the pre-commit
added more non-RHTAP namespaces and renamed the existing data to imitate what is expected in RHTAP.